Add LT_INIT to configure.ac so that autoreconf pulls in recent libtool
auxiliary files.

--- a/configure.ac
+++ b/configure.ac
@@ -24,7 +24,7 @@
 AC_PROG_CC
 
 AC_PROG_INSTALL
-AC_PROG_LIBTOOL
+LT_INIT
 
 dnl if CC is gcc, we can rebuild the dependencies (since the depend rule
 dnl requires gcc).  If it's not, don't rebuild dependencies -- use what was
